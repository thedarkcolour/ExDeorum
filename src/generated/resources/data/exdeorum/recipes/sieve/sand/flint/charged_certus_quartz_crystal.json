{
  "neoforge:conditions": [
    {
      "type": "neoforge:mod_loaded",
      "modid": "ae2"
    }
  ],
  "type": "exdeorum:sieve",
  "ingredient": {
    "item": "minecraft:sand"
  },
  "mesh": "exdeorum:flint_mesh",
  "result": "ae2:charged_certus_quartz_crystal",
  "result_amount": {
    "type": "minecraft:binomial",
    "n": 1.0,
    "p": 0.005
  }
}