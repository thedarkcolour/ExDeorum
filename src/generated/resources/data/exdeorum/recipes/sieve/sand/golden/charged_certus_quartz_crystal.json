{
  "neoforge:conditions": [
    {
      "type": "neoforge:mod_loaded",
      "modid": "ae2"
    }
  ],
  "type": "exdeorum:sieve",
  "ingredient": {
    "item": "minecraft:sand"
  },
  "mesh": "exdeorum:golden_mesh",
  "result": "ae2:charged_certus_quartz_crystal",
  "result_amount": {
    "type": "minecraft:binomial",
    "n": 1.0,
    "p": 0.015
  }
}